zerver/lib/events.py

```diff
@@ -239,11 +240,20 @@ def our_person(p):
                 for key, value in event['data'].items():
                     state['realm_' + key] = value
         elif event['type'] == "subscription":
+            if not include_subscribers and event['op'] in ['peer_add', 'peer_remove']:
+                continue
+
             if event['op'] in ["add"]:
-                # Convert the user_profile IDs to emails since that's what register() returns
-                # TODO: Clean up this situation
-                for item in event["subscriptions"]:
-                    item["subscribers"] = [get_user_profile_by_email(email).id for email in item["subscribers"]]
+                if include_subscribers:
+                    # Convert the user_profile IDs to emails since that's what register() returns
+                    # TODO: Clean up this situation
+                    for item in event["subscriptions"]:
+                        item["subscribers"] = [get_user_profile_by_email(email).id for email in item["subscribers"]]
```

Comment:
It is already IDs in the main list; but I guess you're asking about migrating the event?  Yeah, that's probably worth doing.